test/e2e/middleware-trailing-slash/test/index.test.ts

```diff
@@ -114,15 +114,17 @@ describe('Middleware Runtime trailing slash', () => {
         }
         delete middlewareWithoutEnvs.env
         expect(middlewareWithoutEnvs).toEqual({
-          files: expect.arrayContaining([
-            'server/edge-runtime-webpack.js',
-            'server/middleware.js',
-          ]),
+          files: process.env.TURBOPACK
+            ? expect.toBeArray()
```

Comment:
Should we at least assert a length here?

Reply:
The length can change at any point when the chunking changes and it's not really relevant, we already check if the edge function can run and return the right result.

Reply:
could assert >0